static/app/components/dropdownAutoComplete/menu.tsx

```diff
@@ -133,11 +133,6 @@ export interface MenuProps
    */
   hideInput?: boolean;
 
-  /**
-   * renderProp for the end (right side) of the search input
-   */
-  inputActions?: React.ReactElement;
-
```

Comment:
this was unused, and since it didn’t play well with the refactoring I did, I killed it 💀